<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Assets/PoB.qrc">
    <normaloff>:/Art/Icons/construction.png</normaloff>:/Art/Icons/construction.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="btnBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Application</string>
   </property>
   <widget class="QLabel" name="label_Protocol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Protocol:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>combo_Protocol</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_Proxy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>52</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy Server:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>combo_Proxy</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_BuildPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>82</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Build save path:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineedit_BuildPath</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_NP_Colours">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>112</y>
      <width>120</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Node Power colours:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>combo_NP_Colours</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_Proxy">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>49</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HTTP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SOCKS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SOCKS5H</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_Protocol">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>18</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Auto</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPv6</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_NP_Colours">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>108</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Red &amp; Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red &amp; Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green &amp; Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineedit_BuildPath">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>204</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_Beta">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Opt in to weekly test builds:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_Proxy">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>164</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_BuildPath">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>79</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>380</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Build related</string>
   </property>
   <widget class="QCheckBox" name="check_ShowBuildName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>20</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Show build name in window title</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_ShowThousandsSeparators">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>40</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Show thousands separators</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_ThousandsSeparator">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>65</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BuildPath_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>67</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thousands separator:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineedit_ThousandsSeparator</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineedit_DecimalSeparator">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>95</y>
      <width>30</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_BuildPath_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>97</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal separator:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>lineedit_DecimalSeparator</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_GemQuality">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>127</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default gem quality:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>spin_GemQuality</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_Level">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>157</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default character level:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>spin_Level</cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_Tooltips">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>230</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Show tooltips only for affected slots</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_BuildWarnings">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>210</y>
      <width>210</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Show build warnings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AffixQuality">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>187</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default item affix quality</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buddy">
     <cstring>slider_AffixQuality</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_GemQuality">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>125</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_Level">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>155</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_AffixQuality">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>185</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_AffixQValue">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>187</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nn</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>combo_Protocol</tabstop>
  <tabstop>combo_Proxy</tabstop>
  <tabstop>lineedit_Proxy</tabstop>
  <tabstop>lineedit_BuildPath</tabstop>
  <tabstop>btn_BuildPath</tabstop>
  <tabstop>combo_NP_Colours</tabstop>
  <tabstop>check_Beta</tabstop>
  <tabstop>check_ShowBuildName</tabstop>
  <tabstop>check_ShowThousandsSeparators</tabstop>
  <tabstop>lineedit_ThousandsSeparator</tabstop>
  <tabstop>lineedit_DecimalSeparator</tabstop>
  <tabstop>spin_GemQuality</tabstop>
  <tabstop>spin_Level</tabstop>
  <tabstop>slider_AffixQuality</tabstop>
  <tabstop>check_BuildWarnings</tabstop>
  <tabstop>check_Tooltips</tabstop>
 </tabstops>
 <resources>
  <include location="../../Assets/PoB.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>Config</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
